.github/workflows/jsonrpc.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  RUST_MIN_STACK: "8388608"
```

Comment:
Does the CI ignore `.cargo/config.toml`?

Reply:
Ah, I just saw [590cc9d](https://github.com/deltachat/deltachat-core-rust/pull/3699/commits/590cc9da9fc15620c3cd56767c26fbe0f9c1da36) - if some tooling ignores `.cargo`, then this might be a problem for new devs again. BTW, `cargo nextest run` also ignores `.cargo`, so my go-to solution would actually be putting `#[async_recursion]` on the tests as link2xt proposed (if this works). Then again, maybe it's fine to just merge this, too.

Reply:
hum, I'm not sure what's up with the jsonrpc tests to be honest :(

I'm not too much of a fan of having to modify the code... but if nothing else will do...